patch_scene.py

```diff
@@ -45,7 +45,8 @@ def __patch_scene(scene_id: str,
 
         assert instance not in instance_accumulated_clouds_lookup
 
-        accumulated_point_cloud = point_cloud_accumulator.merge(instance_id=instance,
+        accumulated_point_cloud = point_cloud_accumulator.merge(scene_id=scene_id,
```

Comment:
maybe frame_id could be here too? it would be useful for Gedi

Reply:
Point cloud accumulator merges instance across the entire scene, i.e. across all frames. You have `frame_id` availale inside.